src/sip.c

```diff
@@ -615,6 +634,8 @@ sip_parse_msg_media(sip_msg_t *msg, const u_char *payload)
         if (!strncmp(line, "c=", 2)) {
             if (sscanf(line, "c=IN IP4 %s", dst.ip) && media) {
                 media_set_address(media, dst);
+                strcpy(rtp_stream->dst.ip, dst.ip);
+                strcpy(rtcp_stream->dst.ip, dst.ip);
```

Comment:
I was assuming that a _c=_ string comes before any _m=_ string, so I was storing the ip into `dst.ip` and re-using `dst` struct to fill the port in _m=_ section before creating the stream struct.